modules/nodes-nodes-viewing-listing.adoc

```diff
@@ -209,6 +209,11 @@ Events:     <11>
 <10> The pods on the node.
 <11> The events reported by the node.
 
+[NOTE]
+====
+The control plane label is not automatically added to newly created or updated master nodes. If you want to use the control plane label for your nodes, you can manually configure the label. See xref:../../modules/nodes-nodes-working-updating.adoc[Understanding how to update labels on nodes] for more information about manually configuring labels.
```

Comment:
This xref is wrong. It might pass the build but pretty sure it won't  work.

Reply:
@aireilly Hey, I just got some info from someone else who said we're not supposed to use links in modules (which I vaguely remember from onboarding). I'm going another route with this and will rebase in just a few minutes.